src/octokit.ts

```diff
@@ -2,43 +2,91 @@ import {getInput, warning} from '@actions/core'
 import {Octokit} from '@octokit/action'
 import {retry} from '@octokit/plugin-retry'
 import {throttling} from '@octokit/plugin-throttling'
+import dotenv from 'dotenv'
+import {createAppAuth} from '@octokit/auth-app'
```

Comment:
This is only required when running outside the github actions runners. Should not be committed to the PR.

Reply:
yes, this needs to be commited in PR as this is important file that will establish/entry point of dev env. as we also needs to be taken care about dev env because of open source code.